### PR TITLE
Complete docs for api and readme

### PR DESCRIPTION
**Pull request by tt-loveslife**
Complete readme and api docs

### CODE REVIEW

File: Collision_API.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+### POST /api/v1/crypto/getEncCollision
+
+> 解密碰撞表，因为解密时间是线性的，如果数很大，需要的时间将非常长，通过后台设计碰撞表，将解密时间降低至常量级。
```
Comment (gavinisagi) [inline Collision_API.md]: In English

**Review by gavinisagi — CHANGES_REQUESTED** [review]

File: Collision_API.md (hunk 1 above)
Comment (tt-loveslife) [reply Collision_API.md]: Solved in commit 7222e61

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
...
+
+#6 启动 redis 镜像
+docker run -itd --name redis-test -p 6379:6379 ghcr.io/tygavinzju/redis-sample:1.0.0
+```
+
+## 开发环境
+- go语言安装
+```shell
+wget -c https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local
+# 最好持久化到环境变量
```
Comment (gavinisagi) [inline README.md]: In English

**Review by gavinisagi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tt-loveslife) [reply README.md]: Solved in commit e15e7d2

File: README.md [hunk 3]
```diff
@@ -1 +1,64 @@
-# enc-collision-service
\ No newline at end of file
+# enc-collision-service开发入门
+Windows下开发环境使用`Goland + WSL2 + Docker`体验较好
```
Comment (gavinisagi) [inline README.md:2]: delete it

File: README.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+
+#2 login github docker repo
+export PAT="${GITHUB_TOKEN}"
+echo $PAT | docker login ghcr.io --username ${username} --password-stdin
+
+#3 pull postgres image
+docker pull ghcr.io/tt-loveslife/collisiondatabase:0.0.2
+
+#4 run postgres in docker
+docker run --name collision -p 5432:5432 -e POSTGRES_DB=crypto -e POSTGRES_USER=root -e POSTGRES_PASSWORD=123456  -d ghcr.io/tt-loveslife/collisiondatabase:0.0.2
```
Comment (gavinisagi) [inline README.md:17]: hide db info

**Review by gavinisagi — CHANGES_REQUESTED** [review]

**Review by gavinisagi — APPROVED** [review]

**Pull request merged**